velox/expression/tests/CustomTypeTest.cpp

```diff
@@ -256,7 +258,7 @@ TEST_F(CustomTypeTest, nullConstant) {
 
```

Comment:
Does CustomTypeTest.getCustomTypeNames [test](https://github.com/facebookincubator/velox/blob/main/velox/expression/tests/CustomTypeTest.cpp#L221) need to be updated to use another function? T-Digest isn't returned from [Type.getCustomNames()](https://github.com/facebookincubator/velox/blob/main/velox/type/Type.cpp#L1044). 

Is that expected?

Reply:
We probably are not registering the type factory during test initialization.  That's super minor.